packages/api/cms-api/src/file-utils/files.utils.ts

```diff
@@ -11,8 +11,8 @@ import stream from "stream";
 import { promisify } from "util";
 import { v4 as uuid } from "uuid";
 
-import { type FileUploadInput } from "./dto/file-upload.input";
-import { FilesService } from "./files.service";
+import { FilesService } from "../dam/files/files.service";
```

Comment:
This shouldn't have a dependency on the FilesService if it's meant to be shared between file uploads and DAM.

Reply:
cdc68445d396cc7a95bac542aaf98a32938ef1d3